configure.ac

```diff
@@ -273,11 +273,11 @@ if test -n "$fc_version_info"; then
 fi
 
 # Check if gcc is 11.1 for class(*) select type bug
-AC_MSG_CHECKING([if gcc 11 is loaded])
-if [ test -n "`$FC --version | grep GNU | grep 11\..\..`" ]; then
+AC_MSG_CHECKING([if gcc 11.1.0 is loaded])
+if [ test -n "`$FC --version | grep GNU | grep 11\.1\..`" ]; then
   AC_MSG_RESULT([yes])
-  AC_MSG_ERROR([Compilation with gcc and gfortran 11 is unsupported by this version\
- of FMS. Please use another compiler version])
+  AC_MSG_ERROR([Compilation with gcc and gfortran 11.1.0 is unsupported by this version\
+ of FMS. Please use gcc/gfortran 11.2.0 or an older version.])
```

Comment:
This message is even more confusing because 11.1 is older than 11.2, and when 12 comes out, you will be able to use that.  "Compilation with gcc and gfortran 11.1.0 is unsupported by this version of FMS due to a bug in the compiler. Please use a  different version of gcc/gfortran"